apps/e2e/src/indexer-client/fullSanity.ts

```diff
@@ -331,6 +336,32 @@ async function fullSanity(context: RunContext) {
   });
 
   prettyPrint('Fast Withdrawal Signature', fastWithdrawalSignature);
+
+  const updateLeaderboardRegistrationResult =
+    await client.updateLeaderboardRegistration({
```

Comment:
@yruej301 Will this error after the contest is over?

Reply:
Yes

Reply:
Maybe we shouldn't include it in the sanity then, it'd be flakey

Reply:
Yup, sounds good -- will remove in a follow up pr. Or we can catch the error in case we wanna re-use it in the future